packager/VisibilityChecker.cc

```diff
@@ -30,6 +30,58 @@ static core::SymbolRef getEnumClassForEnumValue(const core::GlobalState &gs, cor
     return core::Symbols::noSymbol();
 }
 
+static void reportMissingTestImport(const core::Context &ctx, const core::packages::PackageInfo &package,
+                                    core::SymbolRef symbol, std::vector<core::LocOffsets> locs) {
+    auto importAdded = false;
+    for (auto it = locs.rbegin(); it != locs.rend(); ++it) {
+        auto loc = *it;
+        if (auto e = ctx.beginError(loc, core::errors::Packager::UsedTestOnlyName)) {
+            auto otherFile = symbol.loc(ctx).file();
+            auto &otherPackage = ctx.state.packageDB().getPackageNameForFile(otherFile);
+            e.setHeader("Used `{}` constant `{}` in non-test file", "test_import", symbol.show(ctx));
+            auto &pkg = ctx.state.packageDB().getPackageInfo(otherPackage);
+            if (!importAdded) {
+                if (auto exp = package.addImport(ctx, pkg, false)) {
+                    e.addAutocorrect(std::move(exp.value()));
+                }
+            }
```

Comment:
`importAdded` is never set to `true`. Can we add a test which would have failed and caught this?